.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
   matrix:
     - LINT_CHECK="1"
     - TESTS="1" ODOO_REPO="OCA/OCB"
-    - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
+    - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="0"
```

Comment:
According to https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L72 should be `MAKEPOT="0"` in the global stage:


```
env:
  global:
    - VERSION="11.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"

```

and should stay `MAKEPOT="1"` in test stage (https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L56)

Reply:
I solved this @astirpe